src/main/java/com/theelixrlabs/healthcare/constants/SecurityConstants.java

```diff
@@ -0,0 +1,10 @@
+package com.theelixrlabs.healthcare.constants;
+
+/**
+ * Constants for spring security implementation
+ */
+public class SecurityConstants {
+    public static final String DEV_PROFILE_CONSTANT = "dev";
+    public static final String GOOGLE_JWK_SET_URI = "https://www.googleapis.com/oauth2/v3/certs";
```

Comment:
I feel line 8 and 9 should be kept in properties file

Reply:
Resolved